jaeger/client.go

```diff
@@ -46,7 +46,7 @@ func NewClient(token string) (*Client, error) {
 			return nil, errParse
 		}
 		timeout := time.Duration(5000 * time.Millisecond)
-		transport, err := httputil.AuthTransport(&auth, &http.Transport{})
+		transport, err := httputil.CreateTransport(&auth, &http.Transport{}, 10*time.Second)
```

Comment:
Still referring to this page https://blog.cloudflare.com/the-complete-guide-to-golang-net-http-timeouts/ (which is becoming my ultimate golang timeout guide :-) )
So this code means we have a client timeout set to 5 seconds (which works for the whole exchange), plus more granular 10 seconds timeouts defined in `CreateTransport` which work for parts of the exchange. I guess these granular ones are useless here, no? (refer to the "Client Timeouts" section of the article, the schema is quite self-explanatory).
Or perhaps, at least if we want this more granular control, the timeout value passed to `CreateTransport` should be less than the 5s defined for the whole?

Reply:
After reading the cloudflare post, I agree with @jotak.
This timeout should be, at most, equal to the Client timeout (in line 53: https://github.com/kiali/kiali/pull/3464/files#diff-31753879fb48a9f42f0390e11ebd20f5be310a8a2f66f1061d3dceb3ff23e517R53).

Reply:
(updated my comment above, +1 for the safety net of having idle timout defined even if it's not necessary at the moment)

Reply:
This is correct, @jshaughn, thanks for catching it.